Extensions/P2P/JsExtension.js

```diff
@@ -0,0 +1,297 @@
+        _('P2P (experimental)'),
+        'JsPlatform/Extensions/p2picon.svg',
+        'JsPlatform/Extensions/p2picon.svg'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/P2P/A_peer.js')
+      .addIncludeFile('Extensions/P2P/B_p2ptools.js')
+      .setFunctionName('gdjs.evtTools.p2p.isReady');
+
+    extension
+      .addCondition(
+        'OnError',
+        _('An error occured'),
+        _('Triggers once when an error occurs.'),
+        _('P2P error occured'),
+        _('P2P (experimental)'),
+        'JsPlatform/Extensions/p2picon.svg',
+        'JsPlatform/Extensions/p2picon.svg'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/P2P/A_peer.js')
+      .addIncludeFile('Extensions/P2P/B_p2ptools.js')
+      .setFunctionName('gdjs.evtTools.p2p.onError');
+    
+    extension
+      .addCondition(
+        'OnDisconnection',
+        _('Peer disconnected'),
+        _('Triggers once when a peer disconnects.'),
+        _('P2P peer disconnected'),
+        _('P2P (experimental)'),
+        'JsPlatform/Extensions/p2picon.svg',
+        'JsPlatform/Extensions/p2picon.svg'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/P2P/A_peer.js')
+      .addIncludeFile('Extensions/P2P/B_p2ptools.js')
+      .setFunctionName('gdjs.evtTools.p2p.onDisconnect');
+
+    extension
+      .addAction(
+        'Connect',
+        _('Connect to another client'),
+        _('Connects the current client to another client using its id'),
+        _('Connect to P2P client _PARAM0_'),
+        _('P2P (experimental)'),
+        'JsPlatform/Extensions/p2picon.svg',
+        'JsPlatform/Extensions/p2picon.svg'
+      )
+      .addParameter('string', _('ID of the other client'), '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/P2P/A_peer.js')
+      .addIncludeFile('Extensions/P2P/B_p2ptools.js')
+      .setFunctionName('gdjs.evtTools.p2p.connect');
+
+    extension
+      .addAction(
+        'UseOwnBroker',
+        _('Connect to own broker server'),
+        _('Connects to your own broker server.'),
+        _('Connect to broker server http://_PARAM0_:_PARAM1_/'),
+        _('P2P (experimental)'),
+        'JsPlatform/Extensions/p2picon.svg',
+        'JsPlatform/Extensions/p2picon.svg'
+      )
+      .addParameter('string', _('Host'), '', false)
+      .addParameter('number', _('Port'), '', false)
+      .addParameter('string', _('Path'), '', false)
+      .addParameter('string', _('Key'), '', false)
+      .addParameter('yesorno', _('SSl enabled?'), '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/P2P/A_peer.js')
+      .addIncludeFile('Extensions/P2P/B_p2ptools.js')
+      .setFunctionName('gdjs.evtTools.p2p.useCustomBrokerServer');
+
+    extension
+      .addAction(
+        'UseDefaultBroker',
+        _('Connect to default broker server'),
+        _('Connects to the default broker server.'),
```

Comment:
[...] server. It is still recommended that you use a custom broker server if your game is published, so that you have full control over it.

Reply:
There is already the hint for that?

Reply:
My bad, perfect then!